ONSdigital/ras-frontstage
# Update Design System to 49.1.0

# What and why?
This PR updates the design system to 49.1.0 from 47.0.0. Although there were a couple of breaking changes there was only one that affected RAS/RM and that being the inclusion of a toggle button for mobile phone usage if there are service links, which we do for both logged in users and those that haven't https://github.com/ONSdigital/design-system/blob/2afa6ae54ae4f9fcdeb02dd45ad469c82693d8f8/src/components/header/_macro-options.md

# How to test?
Firstly check everything looks ok, being new to the team picking anything up less than obvious is like trying to find rocking horse poo. Once done check the new functionality from the design system is working, you can use a mobile or browser stack if you want, but shrinking the screen will be just as good. When the screen is shrunk a link on the top called menu should appear with a dropdown arrow for the service links (make sure they work) both logged in an not

# Trello
https://trello.com/c/bZUk2xwP/1637-update-design-system-to-v49
